presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -55,6 +55,9 @@
 
 public class RuleAssert
 {
+    // 2.5 minute timeout
```

Comment:
for tests, that's a lot

Reply:
fair point.  I'll make it one minute.  Still want to leave lots of room to avoid flaky tests.

Reply:
i would think 10s would be already lots of room. Anyway, since `RuleAssert` creates its own `Rule.Context` and you're replacing `getTimeoutInMillis()` with a void check that throws an exception, this becomes irrelevant. Now, `RuleAssert` can impose no limit on its own, just as it used to do.